_pages/understanding/intro-to-ada.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Introduction to the Americans with Disabilities Act
+permalink: /understanding/intro-to-ada/
+layout: page
+lead: |-
+  The Americans with Disabilities Act (ADA) is a federal civil rights law that prohibits discrimination against people with disabilities in everyday activities. The ADA prohibits discrimination on the basis of disability just as other civil rights laws prohibit discrimination on the basis of race, color, sex, national origin, age, and religion. It guarantees that people with disabilities have the same opportunities as everyone else to enjoy employment opportunities, purchase goods and services, and participate in state and local government programs. 
+
+---
+
+## The ADA Protects People with Disabilities 
+
+A person with a disability is someone who:
+ 
+- has a physical or mental impairment that substantially limits one or more major life activities, 
+- has a history or record of such an impairment (such as cancer that is in remission), or 
+- is perceived by others as having such an impairment (such as a person who has scars from a severe burn).
+ 
+If a person meets this definition, the ADA protects them. Because the ADA is a law, and not a benefit program, you do not need to apply for coverage. 
+ 
+<details>
+  <summary>What does <dfn><i>substantially limits</i></dfn> mean?</summary>
+  <p>The term “substantially limits” is defined broadly and is not meant to be a demanding standard. An example of a condition that is not substantial is a mild allergy to pollen.</p> 
+</details>  
+ 
+<details>
+<summary>What does <dfn><i>major life activities</i></dfn> mean?</summary>
+ 
+<p>Major life activities are the kind of actions that you do every day, including your body’s own internal processes. There are many major life activities in addition to the examples listed here. Some examples include:</p>
```

Comment:
What do you think of using `markdown="1"` on the `<details>` block, which would allow us to use normal markdown formatting for the paragraphs and lists within the blocks?

Reply:
Using markdown would be preferable. 👍

Reply:
The markdown attribute appears to work! I'll refactor this when I have some time.